resources/views/pages/questions/overview.twig

```diff
@@ -21,8 +51,17 @@
                     {% block questions %}
                         {% block faq_text %}
                             {% if has_permission_to('faq.view') %}
-                                <p>{{ __('question.faq_link', [url('/faq')]) | raw }}</p>
+                                {% if not is_admin | default(false) %}
+                                    {{ __('question.faq_link', [url('/faq')]) | raw }}
+                                {% endif %}
+                            {% endif %}
+                        {% endblock %}
+                        {%- block info_text %}
+                            <p>
```

Comment:
Should be inside the if block

Reply:
nope because it looks funny if its not shown

Reply:
then that should be fixed with css and not by adding empty paragraphs

Reply:
😝